### PR TITLE
Remove User Chip on Learner Search Page

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/2875

#### What's this PR do?
removes user chip on mouse hover and added link to learners profile.

#### How should this be manually tested?
Go to http://192.168.99.100:8079/learners/

@pdpinch 
#### Screenshots (if appropriate)
screen shot 2018-03-28 at 7 10 21 pm

### CODE REVIEW

**Comment by aliceriot** [comment]
looks like there is a linting error

File: static/js/components/search/LearnerResult.js [hunk 1]
```diff
@@ -88,23 +59,4 @@ class LearnerResult extends SearchkitComponent {
...
-    setLearnerChipVisibility: (username: ?string): void => {
-      if (ownProps.learnerChipVisibility !== username) {
-        dispatch(setLearnerChipVisibility(username))
-      }
-    }
-  }
-}
-
-export default connect(mapStateToProps, mapDispatchToProps)(LearnerResult)
+export default connect()(LearnerResult)
```
Comment (aliceriot) [inline static/js/components/search/LearnerResult.js]: do we still need `connect()` here? It looks like we're no longer depending on anything from the Redux store

**Comment by amir-qayyum-khan** [comment]
Done @aliceriot

File: static/js/entry/style.js [hunk 2]
```diff
@@ -26,5 +26,3 @@ import "react-telephone-input/css/default.css"
 
 import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css"
 
-// This should come last to override other styles
-import "../../scss/layout.scss"
```
Comment (aliceriot) [inline static/js/entry/style.js]: no, we want to keep this - in `layout.scss` you will just need to delete the import for `user-chip.scss`

Comment (amir-qayyum-khan) [reply static/js/entry/style.js]: oops, fixed @aliceriot

**Pull request merged**